src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -84,25 +84,25 @@ NOTE: Future extensions may redefine the capability format providing they follow
 
 ==== Address
 
-The lower XLEN bits of a capability encode the address of where the capability points.
+The lower XLEN-bits of a capability encode the address of where the capability points.
```

Comment:
```suggestion
The lower XLEN bits of a capability encode the address of where the capability points.
```
It's not used as an adjective here so no need for the hyphen. I went through all of these and fixed them.